bootcamp/tutorials/integration/llama_agents_metadata.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>Nit, if possible, a slightly deeper one-sentence comment can be more helpful, e.g. xxx is needed for printing ... in beautiful format. For this one, it might be nest_asyncio.apply() is needed so that the asyn calls later used for agent ... can work in notebook. you get the idea.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>